### ISSUE TITLE
Missing alt-attributes in fly-out in the right side menu

### ISSUE THREAD

**Issue by AndyScherzinger** [nextcloud/deck#3903]
In the fly-out in the right side menu, there are images without alt-attributes, which must be present for such elements. They are placed in links without link texts, which means that screen reader users don’t get any information on their purpose.

There seems to be a dynamic reading of a text through aria-describeby, but it disappears.

You can use the alternative texts for the images to give these links understandable link texts. Or you can give them alt="" and place texts in the links for much the same effect.

![3](https://user-images.githubusercontent.com/1315170/177596392-ce5ebbde-282c-456e-bd8f-270299db9b4d.jpg)
